### ISSUE TITLE
 Cannot bind argument to parameter 'OutDriveDenom' because it is null.

### ISSUE THREAD

**Issue by realdevnullius** [tydeno/Ploto#77]
`PS C:\Windows\system32> Start-PlotoSpawns
PlotoManager @ 2-6-2021 19:35:43 : Ploto Manager started.
PlotoManager @ 2-6-2021 19:35:43 : Loading config from  C:\Users\Gebruiker\.chia\mainnet\config\PlotoSpawnerConfig.json...
PlotoManager @ 2-6-2021 19:35:43 : InputAmountToSpawn: 100
PlotoManager @ 2-6-2021 19:35:43 : Intervall to wait: 5
PlotoManager @ 2-6-2021 19:35:43 : PlotoFy is set to startup. Checking for active PlotoFy jobs...
PlotoManager @ 2-6-2021 19:35:43 : We have found active PlotoFy jobs. Stopping it and starting fresh...

Id     Name            PSJobTypeName   State         HasMoreData     Location             Com
                                                                                          man
                                                                                          d
--     ----            -------------   -----         -----------     --------             ---
9      PlotoFy         BackgroundJob   Running       True            localhost            ...
PlotoManager @ 2-6-2021 19:35:44 : Started PlotoFy successfully. Check your Discord
---------------------------------------------------------------------------------
Invoke-PlotoJob : Cannot bind argument to parameter 'OutDriveDenom' because it is null.
At C:\Users\Gebruiker\Downloads\Ploto\Ploto.psm1:1106 char:104
+ ... ize $BufferSize -Thread $Thread -OutDriveDenom $OutDriveDenom -TempDr ...
+                                                    ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Invoke-PlotoJob], ParameterBindingValidation
   Exception
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Invoke-PlotoJob

Invoke-PlotoJob : Cannot bind argument to parameter 'OutDriveDenom' because it is null.
At C:\Users\Gebruiker\Downloads\Ploto\Ploto.psm1:1106 char:104
+ ... ize $BufferSize -Thread $Thread -OutDriveDenom $OutDriveDenom -TempDr ...`

I know I'm tired a.f. but what am I doing wrong here?

**Comment by realdevnullius** [comment]
This is part of my config file, am I doing this right?

`  "DiskConfig": [
    {
      "D": "plot",
      "C": "",
      "E": "out",
      "EnableT2": "false"
    }
  ],
`

**Comment by Sp4rkR4t** [comment]
![image](https://user-images.githubusercontent.com/3926906/120532710-dd1d4500-c3d7-11eb-8b16-32dba1bdb195.png)

It should look more like this, you seem to have added in the disk drive letters which is not needed, ploto will scan for drives including the chosen TempDrive name or OutDrive name, for example if you have a drive named ChiaStore and the OutDrive demon is set to "store" the ChiaStore drive will be used for storing the plots.

**Comment by realdevnullius** [comment]
Thanks Tydeno on Discord, and here!

I'm really slow this night but got it working... For other newbies, no stress! It's simple.

Tydeno Ploto will look for Disk Label patterns. You can set those label patterns in the `C:\Users\YourUserName\.chia\mainnet\config\PlotoSpawnerConfig.json` file's "DiskConfig": section.

I use my NVME boot disk as a secondary temp (-2). Its disk label is "Lokale Schijf".
I also have an SSD disk, labeled SSD00 and a storage disk, named "HP_HDD_00". So my DiskConfig section ends up looking like this:
```
  "DiskConfig": [
    {
      "TempDriveDenom": "SSD",
      "Temp2Denom": "Lokale schijf",
      "OutDriveDenom": "HP_HDD",
      "EnableT2": "true"
    }
  ],

Nothing more needs to be edited, you are ready to Ploto away ;-)
```

**Comment by realdevnullius** [comment]
Update:

temp2 appeared not be working properly yet. When I started Ploto, I got:
 
`T2Drive         : None`

I wanted to use my NVMe system disk as a secondary temp directory. By default, my C:-drive's label was set to "Lokale schijf".

I tried setting the denom to "Lokale" instead, but that didn't work either.

The solution was renaming the system volume to something else, I choose "windows' and ended up with this DiskSection:
```
  "DiskConfig": [
    {
      "TempDriveDenom": "SSD",
      "Temp2Denom": "windows",
      "OutDriveDenom": "HP_HDD",
      "EnableT2": "true"
    }
  ],

Closing it again :)
```
